nbviewer/templates/layout.html

```diff
@@ -19,6 +19,16 @@
 {%- endmacro %}
 
 
+{% macro head_js_icon(js, name, icon, accesskey=None) -%}
+  <li>
+    <a href="javascript:{{js}}" title="{{name}}" {%if accesskey %}accesskey={{accesskey}}{% endif %}>
+      <span class="fa fa-{{icon}} fa-2x menu-icon"></span>
+      <span class="menu-text">{{name}}</span>
```

Comment:
I like this, but it might be worth expanding the totality of the js that's included…

maybe something like (note this won't work and may not be valid js

```javascript
<script>
for (x in $.('hide_input_cells')){
   $(x.id).height($(x.id).height())
}
</script>
```

Or should this go below since you're actually passing the javascript _in_ there and there is minimal js in this actual macro…

NB: I've not seen this use of an `href` or `a` before, I'm intrigued.